tests/utils.py

```diff
@@ -58,62 +58,68 @@ def run_with_reference(
         ):
 
 
-    res, config = execute_pysvzerod(test_config, "solver")
+    res = pysvzerod.simulate(test_config)
```

Comment:
Use `execute_pysvzerod` as before. This function can switch between Python and binary execution, which is necessary for code coverage